networktest.go

```diff
@@ -53,6 +54,13 @@ var (
 	defaultClientPort = 19556
 
 	harnessNetParams = &chaincfg.SimNetParams
+
+	// nodeOutput[0|1] is a flag that can be set to pass the log output from the
```

Comment:
This should be further generalized, current tests may have up to four distinct nodes active at a given time.